suite-native/graph/src/components/TransactionEventTooltip.tsx

```diff
@@ -44,20 +54,74 @@ const TooltipCardStyle = prepareNativeStyle(utils => ({
     paddingVertical: 1.5 * utils.spacings.small,
 }));
 
-const EventTooltipRow = ({ title, signValue, networkSymbol, value }: EventTooltipRowProps) => (
+const TokenAmountTooltipFormatter = ({
+    accountKey,
+    tokenAddress,
+    networkSymbol,
+    value,
+}: {
+    accountKey: AccountKey;
+    tokenAddress: TokenAddress;
+    networkSymbol: NetworkSymbol;
+    value: number;
+}) => {
+    const tokenInfo = useSelector((state: AccountsRootState) =>
+        selectEthereumAccountTokenInfo(state, accountKey, tokenAddress),
+    );
+
+    const tokenSymbol = tokenInfo?.symbol?.toUpperCase() as TokenSymbol;
+    const tokenDecimals = tokenInfo?.decimals;
+
+    if (!tokenSymbol || !tokenDecimals) {
+        return null;
+    }
+
+    // We might want to add support for other networks in the future.
+    if (getNetworkType(networkSymbol) === 'ethereum') {
```

Comment:
nit: What about placing this condition to the preceding if statement like:
```
  if (!tokenSymbol || !tokenDecimals || !getNetworkType(networkSymbol) === 'ethereum'){
    return null
  }
```

Then you could remove this `if` statemenet and also delete the `return null` at the end of this function.

Reply:
I think this version is better because there might be more than ETH tokens so you will only add more `else if` branches for that.